app/views/admin/user_groups/_form.haml

```diff
@@ -1,4 +1,15 @@
 .row
-  .col-8
+  .col-sm-6
     = f.input :name
+
     = f.input :user_ids, as: :select_two, label_method: :name, collection: User.active.not_system, label: 'Users', input_html: { multiple: true }
+  .col-sm-6
+    .well.mt-6
+      %p
+        It is often helpful to name this group for the agency where the people included in the group work and their role in that agency.  Some examples:
```

Comment:
This helper text doesn't show up on the page for adding a new user group in HMIS. Is there a way that warehouse and hmis can share the same template for this?